### PR TITLE
#1952 Improvements to continuous integration

### LINKED ISSUE #1952: Improvements in nightly release
- rename `ci-release.yml` in `nightly-release.yml`
- use distrib, distrib-omc and distrib-headers instead of zipping the deploy folder
- store the 3 versions mentionned above in the assets
- make nightly release a draft to avoid having it visible first in the welcome page of dynawo

### PR DESCRIPTION
**Pull request by zamarrenolm**
Closes #1952

### CODE REVIEW

**Comment by gautierbureau** [comment]
Another issue is that currently the tag nightly is never updated. Maybe we need to delete it before creating the release very night. Maybe it can be done in another PR

**Comment by gautierbureau** [comment]
And it seems that the upload is now failing for some reason https://github.com/dynawo/dynawo/actions/runs/1423802129

**Comment by zamarrenolm** [comment]
> And it seems that the upload is now failing for some reason https://github.com/dynawo/dynawo/actions/runs/1423802129

It seems related to failures in the connection with github servers. The issue is described in https://github.com/actions/upload-artifact/issues/116. The upload of artifacts retries with exponential back-off, to reduce the likelihood of these failures.

I've manually re-run the jobs to see if it was a temporary error.

**Comment by zamarrenolm** [comment]
> Another issue is that currently the tag nightly is never updated. Maybe we need to delete it before creating the release very night. Maybe it can be done in another PR

We will verify this. It has also been noted by @lecourtoisflo.

File: .github/workflows/tools.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
+greet()
```
Comment (gautierbureau) [inline .github/workflows/tools.sh:1]: function is not used

**Review by gautierbureau — CHANGES_REQUESTED** [review]

**Comment by gautierbureau** [comment]
Could you squash the commits after your correction and then I will merge it

**Comment by zamarrenolm** [comment]
> Could you squash the commits after your correction and then I will merge it

Squash and rebase on current master ?

**Comment by zamarrenolm** [comment]
Commits squashed

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**